CHANGELOG.md

```diff
@@ -795,8 +797,8 @@ You'll need to have [husky](https://www.npmjs.com/package/husky) installed for t
 ### 2.0.0-beta.2
```

Comment:
Note that I didn't touch this, I'm guessing when Prettier ran on the commit it made these formatting adjustments!

Reply:
No worries, that's fine